index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+</head>
+
+<body>
+  <header id="brand" class="toolbar">
+    <div class="my-logo">
+      Richard Burneo
+    </div>
+    <nav class="menu-botton">
+      <button><i class="fa fa-bars fa-spin"></i></button>
+    </nav>
+  </header>
+  <section class="headline">
+    <div class="information">
+      <h1 class="my-name">Hey there. <br>I'm Richard</h1>
+      <h2 class="iam">I'm Sotfware Developer</h2>
+      <p class="my-description">I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div class="social-network">
+      <ul>
+        <li>
+          <a href="#" class="linkedin">
+            <i class="fa-brands fa-linkedin"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="facebook">
+            <i class="fa-brands fa-facebook"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="github">
+            <i class="fa-brands fa-github"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="twitter">
+            <i class="fa-brands fa-twitter"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="Instagram">
+            <i class="fa-brands fa-instagram"></i>
+          </a>
+        </li>
+
+
+      </ul>
+    </div>
```

Comment:
- [ ] Kindly add the proper background image (decorations are always background-images, not `img` tags). 
- [ ] KIndly style the social media icons to follow the design template, they are currently still taking on the `a` tag properties.

 _See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/164888931-09b8b022-1117-40ba-b2d6-eadb8fd16888.png) | ![image](https://user-images.githubusercontent.com/49621279/164888964-9316805c-9908-40e5-b873-7a2987f07731.png)

Reply:
I did it, but I have problems with the size background image.